lib/Field/FieldMiddle.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+import Col from 'react-bootstrap/lib/Col';
+
+export const FieldMiddle = ({ children, className }) => (
```

Comment:
Similar feedback to `FieldLeft` there are possible names changes here;

- `FieldBody`
- `FieldContent`
- `FieldEditor`

What do you think?

Reply:
Dismiss this feedback. Too much effort to name things 😂